src/Common/tests/System.Xml.XPath/CoreFunctionLibrary/StringFunctionsTests.cs

```diff
@@ -529,6 +529,34 @@ public static void StringFunctionsTest2436()
 
         /// <summary>
         /// Verify result.
+        /// normalize-space("") = ""
+        /// </summary>
+        [Fact]
+        public static void StringFunctionsTest2473()
```

Comment:
Sounds good, whatever test that proves that we don't change the existing behavior is fine.